juju/juju
# [JUJU-466] Remove bridge-utils installation from cloud-init

We don't need bridge-utils, so we can save a little on initialisation by not installing it.

The old setup script for venerable versions of LXD is no longer required and removed.

A couple of unused testing charms using the old LXD script are removed.

## QA steps

I tested on MAAS. For each of focal, bionic, xenial and trusty:
- `juju add-machine --series <series>`.
- `juju add-machine lxd:<machine id from above> [space constraint if required]`
- Interfaces should be bridged and container machine created.

## Documentation changes

None.

## Bug reference

N/A
